src/locales/en.yaml

```
@@ -168,6 +169,7 @@ app:
       upload_print: Upload & Print
       view: View
       reset_stats: Reset Stats
+      home_all: HomeAll
```

```suggestion
      home_all: All
```

though i don't really see a need to add another key just for this case..

---

as mentioned above, `All` is used for all_camera and home_all, I wanna to keep the style consistent with `homeX` `homeY` `homeAll`(insteda of `All`) when translated into Chinese, they are just ok in English, but need to distinguish in Chinese, maybe in others language.